test/controllers/static_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class StaticControllerTest < ActionController::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
Is there anything we can reasonably test for the static controller? If so, we should do so…